script/release.sh

```diff
@@ -71,6 +71,7 @@ clone() {
   cd $CLI
   git remote remove origin
   git remote add origin git@github.com:$ORIGIN_REPO/$CLI.git
+  git remote add upstream git@github.com:$UPSTREAM_REPO/$CLI.git
```

Comment:
Ahhhhh, I get it. So https for upstream and ssh for origin and then we have the best of both-worlds.